tensorflow/java/src/test/java/org/tensorflow/DirectSessionTest.java

```diff
@@ -0,0 +1,134 @@
+/* Copyright 2020 Elphel, Inc.
+
+SPDX-License-Identifier: Apache-2.0
+==============================================================================*/
+
+package org.tensorflow;
+
+import static org.junit.Assert.assertEquals;
+
+import java.nio.FloatBuffer;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class DirectSessionTest {
+  private static final float EPSILON_F = 1e-7f;
+
+  @Test
+  public void DirectSessionTest() {
+    try (Graph g = new Graph();
+        Session s = new Session(g)) {
+
+      Operation input1 = g.opBuilder("Placeholder","input1")
+                          .setAttr("dtype",DataType.FLOAT)
+                          .build();
+
+      Operation input2 = g.opBuilder("Const", "input2")
+                          .setAttr("dtype", DataType.FLOAT)
+                          .setAttr("value", Tensor.<Float>create((float) 2.0, Float.class))
+                          .build();
+
+      Operation output = g.opBuilder("Mul", "output1")
+                          .addInput(input1.output(0))
+                          .addInput(input2.output(0))
+                          .build();
+
+      int size = 256;
+
+      long[] shape = {size};
+      float[] floats = new float[size];
+      float[] expected = new float[size];
+
+      for (int i = 0; i < size; ++i) {
+        floats[i] = i+1;
+        expected[i] = 2*floats[i];
+      }
+
+      Tensor<Float> t_cpu = Tensor.create(shape,FloatBuffer.wrap(floats));
+
+      // dummy run
+      Tensor<?> res_cpu = s.runner().fetch("output1").feed("input1", t_cpu).run().get(0);
```

Comment:
Lifetime of the created and resulting tensors must be managed by a `try-with-resource` block.